_episodes/06-files.md

```diff
@@ -85,7 +85,7 @@ inflammation-01.csv
 ~~~
 {: .output}
 
-![Analysis of inflammation-01.csv](../fig/03-loop_49_1.png)
+![Output from the first iteration of the for loop. Three line graphs showing the daily average, maximum and minimum inflammation over a 40-day period for the patients in inflammation-01.csv.](../fig/03-loop_49_1.png)
 
```

Comment:
In the lesson we try to keep the line lengths under 100 characters. I think we could split this into multiple lines like:
```
![Output from the first iteration of the for loop. Three line graphs showing the daily average,
maximum and minimum inflammation over a 40-day period for the patients in inflammation-01.csv.](
../fig/03-loop_49_1.png)
```

Reply:
Sounds good

Reply:
Ok :). If you wouldn't mind making that change when you find the time, that would be great. Thank you!

Reply:
Done, let me know if you are happy